libbeat/service/service.go

```diff
@@ -44,10 +44,17 @@ func HandleSignals(stopFunction func(), cancel context.CancelFunc) {
 
 	// On ^C or SIGTERM, gracefully stop the sniffer
```

Comment:
Minor, but this comment is now out of date (and has been for a while). Perhaps: `// On termination signals, gracefully stop the Beat`